packages/suite/src/support/messages.ts

```diff
@@ -2076,7 +2076,7 @@ const definedMessages = defineMessages({
         id: 'TR_YOUR_CURRENT_FIRMWARE',
     },
     TR_YOUR_CURRENT_VERSION: {
-        defaultMessage: 'Your current Suite version is "{version}"',
+        defaultMessage: 'Your current Suite version is',
```

Comment:
previously version was part of translation. this might be needed in some languages that need to move word order. is there any reason why <VersionLink /> can not be part of translation?

Reply:
Hmm thinking of it we actually want to do it like you did, possibly with ":". This way it will be nicely unified with version at the end disregard of chosen language. 
`आपका वर्शन है: 05.13.2020.BETA` looks in this case better than `आपका वर्शन 05.13.2020.BETA है`

Reply:
Even if we kept the version variable there, it doesn't mean it has to be translated to local lingo. But I am fine with this change as it looks like the version is separated in a button.

Reply:
Oh it is not completely separated, in this case I would argue for keeping the `version` variable